linchpin/rundb/drivers.py

```diff
@@ -1,13 +1,24 @@
 from __future__ import absolute_import
 
 from .tinydb import TinyRunDB
-from .mongodb import MongoDB
 
+try:
+    from .mongodb import MongoDB
+except Exception as e:
+    print(str(e))
+    print("pymongo not installed")
```

Comment:
Can you change this error message to note that pymongo is not available in python 2?  Also, can you provide some indication that this is a warning?

Reply:
Sure @14rcole I will update it